tests/tests_suite_test.go

```diff
@@ -173,6 +177,18 @@ func (s *newSspStrategy) SkipSspUpdateTestsIfNeeded() {
 	// Do not skip SSP update tests in this strategy
 }
 
+func (s *newSspStrategy) SkipUnlessSingleReplicaTopologyMode() {
+	if topologyMode != osconfv1.SingleReplicaTopologyMode {
+		Skip("Tests that are specific for HighlyAvailableTopologyMode are disabled", 1)
+	}
+}
+
+func (s *newSspStrategy) SkipUnlessHighlyAvailableTopologyMode() {
```

Comment:
This method is not used anywhere. If you don't plan to use it in a future PR, can you remove it?

Reply:
I am actually planning to merge this PR only after the SNO lane will be added and then I will need this method. If I will not use it I will remove it. Thank you for noticing.

Reply:
Ok